src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,10 @@
+
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+   id: root => root._id,
+   birthday: root => new Date(root.birthday).toLocaleDateString(),
```

Comment:
you GraphQL type should be `DateTime` so here you don't even have to do that as it would expect a Date object to begin with